core/src/main/java/org/everit/json/schema/loader/SchemaLoader.java

```diff
@@ -404,8 +423,13 @@ public SchemaLoader(SchemaLoaderBuilder builder) {
         }
     }
 
+    private BooleanSchema.Builder buildBooleanSchema() {
```

Comment:
I think now you can remove this method (just return `BooleanSchema.builder()` in line 414)